spec/services/signature_page_service_spec.rb

```diff
@@ -3,23 +3,15 @@
 require 'rails_helper'
 
 RSpec.describe SignaturePageService do
+  let(:dissertation_path) { ActiveStorageSupport.filepath_for_blob(submission.dissertation_file.blob) }
   let(:etd_type) { 'Thesis' }
   let(:submission) do
-    create(:submission, :submittable, :with_readers, :with_advisors, :with_supplemental_files, etd_type:)
-  end
-  let(:temp_dir) { Dir.mktmpdir }
-  let(:dissertation_path) { File.join(temp_dir, 'dissertation.pdf') }
-
-  before do
-    FileUtils.cp(file_fixture('dissertation.pdf'), dissertation_path)
-  end
-
-  after do
-    FileUtils.rm_rf(temp_dir)
```

Comment:
The test should give us more confidence now that it works closer to how the code will in practice.